src/models/Stats/index.js

```diff
@@ -1,28 +1,33 @@
 // @flow
+export type StatsConstructor = {
+  missingTranslationsCount: number,
+  newKeysCount: number,
+  completePercentage: number,
+  translationKeysCount: number,
+  localesCount: ?number,
+  projectsCount: ?number
+}
+
 class Stats {
   missingTranslationsCount: number
   newKeysCount: number
   completePercentage: number
+  translationKeysCount: number
   projectsCount: number
   localesCount: number
 
-  constructor(
-    missingTranslationsCount: number,
-    newKeysCount: number,
-    completePercentage: number,
-    localesCount: ?number,
-    projectsCount: ?number
-  ) {
-    this.missingTranslationsCount = missingTranslationsCount
-    this.newKeysCount = newKeysCount
-    this.completePercentage = completePercentage
+  constructor(args: StatsConstructor) {
+    this.missingTranslationsCount = args.missingTranslationsCount
+    this.newKeysCount = args.newKeysCount
+    this.completePercentage = args.completePercentage
+    this.translationKeysCount = args.translationKeysCount
 
-    if (localesCount) {
-      this.localesCount = localesCount
+    if (args.localesCount) {
+      this.localesCount = args.localesCount
```

Comment:
again, it’s fine setting them to undefined, the `if(…)` guards are not really necessary.